### PR TITLE
Hydrogen sensor

### PR DESCRIPTION
**Pull request by darshanjain12**

### CODE REVIEW

File: ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml [hunk 1]
```diff
@@ -103,4 +103,9 @@ SSS:
   is_abstract: true
   implements:
   - /SENSOR
-  
\ No newline at end of file
+
+SENSOR:
```
Comment (cstirdivant) [inline ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml:107]: You do not need to define this here. The `/SENSOR` in the other yaml should suffice.

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml (hunk 1 above)
Comment (darshanjain12) [reply ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml:107]: Ohkay Thanks. Done the required changes.

Comment (cstirdivant) [reply ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml:107]: Please remove the SENSOR type from this file.

File: ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml [hunk 2]
```diff
@@ -103,4 +103,9 @@ SSS:
   is_abstract: true
   implements:
   - /SENSOR
-  
\ No newline at end of file
+
+HDS:
+  description: "Hydrogen Sensor ."
```
Comment (tasodorff) [inline ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml:108]: nit: remove whitespace ("Hydrogen Sensor.")

**Review by tasodorff — APPROVED** [review]
minor nit; approving prior to fix.

File: ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml (hunk 2 above)
Comment (darshanjain12) [reply ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml]: DONE

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**